This ensures that go will use the locally installed toolchain, and is the
recommended setting for distributions according to upstream.

--- a/go.env      2024-12-13 14:45:14.618692811 +0000
+++ b/go.env      2024-12-13 14:46:20.148353559 +0000
@@ -9,4 +9,4 @@

 # Automatically download newer toolchains as directed by go.mod files.
 # See https://go.dev/doc/toolchain for details.
-GOTOOLCHAIN=auto
+GOTOOLCHAIN=local
